scripts/Dockerfile.devel

```diff
@@ -1,11 +1,13 @@
-FROM ceph/ceph:v14
+FROM ceph/ceph:v15
 
-ENV GOPATH=/go
+ENV GOPATH=/go \
+	GO111MODULE=on
 
 RUN yum -y install \
 	golang \
+	git \
 	make \
 	librados-devel \
 	librbd-devel \
-    && yum -y update \
-    && true
+	&& yum -y update \
```

Comment:
why did you change the indention? This removes the distinction between the commands. Please correct the indention again

Reply:
it was by mistake fixed now. PTAL